frontend/packages/console-dynamic-plugin-sdk/src/utils/k8s.ts

```diff
@@ -0,0 +1,17 @@
+import { K8sKind } from '@console/internal/module/k8s/types';
+
+/** get K8sKind from extension's model */
+export const getK8sKindFromExtension = ({
```

Comment:
This shouldn't be in the SDK. It's specific to our current implementation of `K8sKind`.

Maybe even change the signature of `referenceForModel`:

```
export const referenceForModel = (model: K8sKind | { group: string, version: string, kind: string}): GroupVersionKind =>
```

Reply:
but `k8sKind` has different names for the fields. `apiGroup` and `apiVersion`

Reply:
we can maybe create another `referenceForExtensionModel`